interfaces/builtin/connectivity_observe.go

```diff
@@ -0,0 +1,156 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var connectivityObservePermanentSlotAppArmor = []byte(`
+# Description: Allow owning the NetworkingStatus bus name on the system bus
+
+# DBus accesses
+#include <abstractions/dbus-strict>
+
+dbus (send)
+   bus=system
+   path=/org/freedesktop/DBus
+   interface=org.freedesktop.DBus
+   member={Request,Release}Name
+   peer=(name=org.freedesktop.DBus),
+
+dbus (bind)
+   bus=system
+   name="com.ubuntu.connectivity1.NetworkingStatus",
+`)
+
+var connectivityObserveConnectedSlotAppArmor = []byte(`
+# Description: Allow access to NetworkingStatus service.
+
+dbus (receive)
+    bus=system
+    path=/com/ubuntu/connectivity1/NetworkingStatus{,/**}
+    interface=org.freedesktop.DBus.*,
+`)
+
+var connectivityObserveConnectedPlugAppArmor = []byte(`
+# Description: Allow using NetworkingStatus service.
+
+#include <abstractions/dbus-strict>
+
+# Allow all access to NetworkingStatus service
+dbus (receive, send)
```

Comment:
Should this be only 'send'? I ask because you only have 'recv' in connectivityObserveConnectedSlotAppArmor and replies are automatically allowed.